test/test_plugin_helper.rb

```diff
@@ -6,3 +6,21 @@
 # Add plugin to FactoryGirl's paths
 FactoryGirl.definition_file_paths << File.join(File.dirname(__FILE__), 'factories')
 FactoryGirl.reload
+
+# Foreman's setup doesn't handle cleaning up for Minitest::Spec
+DatabaseCleaner.strategy = :transaction
+
+class Minitest::Spec
+  class << self
+    alias_method :context, :describe
+  end
+
+  before :each do
+    DatabaseCleaner.start
+    Setting::RemoteExecution.load_defaults
+  end
+
+  after :each do
+    DatabaseCleaner.clean
```

Comment:
Also if you notice above I have ` alias_method :context, :describe`, otherwise context isn't defined in the extensions tests...and I get an exception.

I've never used `Minitest::Spec` before, is there some rails magic that's maybe not working with testing the concerns?